hack/test/spider/main.go

```diff
@@ -0,0 +1,203 @@
+package main
```

Comment:
Could we have a comment or some --help text or something that says what this file actually does.